gradle/dependencies.gradle

```diff
@@ -183,7 +183,7 @@ ext {
       jacoco               : '0.2',
       googleServices       : '4.3.3',
       mapboxSdkVersions    : '1.1.3',
-      dokka                : '1.4.20',
+      dokka                : '1.6.21',
```

Comment:
in previous versions I checked (~1.5.x) was changed entry point from `_modules.html` to another file. Could you check if it's true for this version? If so, we need support from the `docs team` to support these changes

Reply:
Going to merge here and we're going to test publishing the docs in `staging` / locally so we can confirm that everything is working fine before next week's release 🚀

Reply:
let's wait for android-docs PR to be merged 🙏 and then merge this one